semgrep/semgrep/app/session.py

```diff
@@ -127,7 +127,7 @@ def authenticate(self) -> None:
         self.token = auth.get_token()
 
         metrics = get_state().metrics
-        metrics.set_is_authenticated(bool(self.token))
+        metrics.add_sanitized_token(self.token)
```

Comment:
makes it seem like self.token is santized?

Reply:
good point, my thinking was "add token after sanitizing it", but I see how I need a new naming scheme to make this unambiguous 🤔